src/backend/config.ts

```diff
@@ -18,7 +18,7 @@ const Environment = () => {
             buildPath: 'frontend_development',
             namespace: 'local',
             proxyUrl: 'https://familie-ba-sak.intern.dev.nav.no',
-            familieTilbakeUrl: 'https://familie-tilbake.intern.dev.nav.no',
+            familieTilbakeUrl: 'https://familie-tilbake-frontend.intern.dev.nav.no',
```

Comment:
Denne URLen lå inne feil for lokalt-mot-preprod. Nå har den fått samme verdi som for preprod 😊